src/AppBundle/Tests/Entity/PlaceTest.php

```diff
@@ -12,6 +12,41 @@
 
 class PlaceTest extends PHPUnit_Framework_TestCase {
     
+    /**
+     * @dataProvider GetNameData
+     */ 
+    public function testGetName($testPlace) {
+        
+        $place = new Place();
+        $place->setName($testPlace);
+        $this->assertEquals($testPlace, $place->getName());
+    }
+    
+    public function getNameData(){
+        
+        return array(
+            array("Vancouver//"),
+            array("[Vancouver]"),
+            array("Va?ncouver"),
+            array("**Vancouver"),
+            array("^Vancouver")
+            
```

Comment:
This is interesting! It would be useful to test that getName() removed question marks, commas, and spaces at the start of a place name. Could you add that please?

Reply:
Strange, just those assertions keep failing. I'm not sure why...